smart_campus/app/templates/app/station_new.html

```diff
@@ -121,18 +108,18 @@
 {% block script %}
   {{ block.super }}
   <script>
-  /*Get location(lat, lng) of beacons, 
-    In order to show them on the Map*/
-  var location_arr = [];
-  
-  {% if beacons %}
-    {% for beacon in beacons %}
-      location_arr.push(['{{ beacon.name }}', {{ beacon.location.y }}, {{ beacon.location.x }}])
-    {% endfor %}
-  {% endif %}
+        var location_arr = [];
```

Comment:
Indentation.

Reply:
okay.
But it was the last PR's content, should I indent it?

Reply:
okay, removed it.